apps/builder/app/builder/features/ai/ai-fetch-result.ts

```diff
@@ -191,10 +191,15 @@ export const fetchResult = async (
 const $availableComponentsNames = computed(
   [registeredComponentMetasStore],
   (metas) => {
-    const exclude = ["Body", "Slot"];
+    const exclude = [
+      "Body",
+      "Slot",
+      // @todo Remove Radix exclusion when the model has been fine-tuned to understand them.
+      "@webstudio-is/sdk-components-react-radix:",
+    ];
 
     return [...metas.keys()]
-      .filter((name) => !exclude.includes(name))
+      .filter((name) => !exclude.some((excluded) => name.startsWith(excluded)))
```

Comment:
- We discussed the disabling should only happen for prod, we need this for testing on preview behind flag
- formatting: no !somethig

Reply:
please use feature flag, all features enabled in preview